website/templates/profile/oauth_app_detail.mako

```diff
@@ -0,0 +1,93 @@
+<%inherit file="base.mako"/>
+<%def name="title()">Application Detail</%def>
+<%def name="content()">
+<style type="text/css">
+    .border-box {
+        padding: 15px 10px;
+        border-radius: 4px;
+        border: solid #DDD;
+        border-width: 1px 0;
+        line-height: 1.1;
+        display: block;
+        margin-bottom: 1em;
+    }
+</style>
+
+<h2 class="page-header">OAuth Application Settings</h2>
+
+<div class="row">
+
+    <div class="col-sm-3">
+        <div class="panel panel-default">
+            <ul class="nav nav-stacked nav-pills">
+                <li><a href="${ web_url_for('user_profile') }">Profile Information</a></li>
+                <li><a href="${ web_url_for('user_account') }">Account Settings</a></li>
+                <li><a href="${ web_url_for('user_addons') }">Configure Add-ons</a></li>
+                <li><a href="${ web_url_for('user_notifications') }">Notifications</a></li>
+                <li><a href="${ web_url_for('oauth_application_config') }">Developer apps</a></li>
+            </ul>
+        </div><!-- end sidebar -->
+    </div>
+
+    <div class="col-sm-9 col-md-7">
+
+        <div id="app-detail">
+            <div id="app-keys" class="border-box text-right text-muted"
+                 data-bind="visible: dataUrl">
+                <p><span><strong>Client ID</strong>:</span> <br><span data-bind="text: content().clientId"></span></p>
+                <p><span><strong>Client secret</strong>:</span> <br><span data-bind="text: content().clientSecret"></span></p>
+            </div>
+            <div id="app-fields">
+                <!-- TODO: Add revoke/ reset buttons -->
+                <form role="form">
```

Comment:
It looks like you could bind this with: content and focus all of your data-binds. The only thing you're using outside of context that you aren't already calling absolutely is the dataurl. http://knockoutjs.com/documentation/with-binding.html

This is not necessarily good advice, knockout being what it is, but I figured I'd mention.